<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>781</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 8.08.09 AM.jpeg);</string>
     </property>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>620</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/Screenshot 2024-06-27 012516.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>520</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 8.08.11 AM.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>440</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 8.08.10 AM.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>700</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 8.21.18 AM.jpeg);
</string>
     </property>
     <widget class="QLabel" name="turnLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
 </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>290</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_home">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 4.35.35 PM.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_restart">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/WhatsApp Image 2024-06-26 at 4.23.45 PM.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>290</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>440</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>450</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
    border: 6px rgb(63, 124, 171);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page4">
     <widget class="QLabel" name="label_losses">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Losses</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_wins">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>280</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>wins :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_draws">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="gameDetailsLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>330</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>gameDetailsLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_userid">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_wins">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_draws">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Draws:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>User name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_losses">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_show">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_user_id">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>User id :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_total_game">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Game:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_enter_the_number">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>380</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_totalgame">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Home_Profile">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTicTacToe">
    <property name="title">
     <string>TicTacToe</string>
    </property>
   </widget>
   <addaction name="menuTicTacToe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
